images/win/Windows2016-Azure.json

```diff
@@ -89,6 +89,11 @@
             "source": "{{ template_dir }}/scripts/Tests",
             "destination": "{{user `image_folder`}}"
         },
+        {
+            "type": "file",
+            "source": "{{ template_dir }}/scripts/Tests-Windows2016",
```

Comment:
Stop creating separate folders. Use the unified one and helper function Test-IsWin16/19